Makefile

```diff
@@ -1,15 +1,29 @@
-export TARGET = iphone:clang:latest:6.0
-INSTALL_TARGET_PROCESSES += SpringBoard
+DEBUG = 0
 
-#DEBUG=1
+PACKAGE_VERSION = $(THEOS_PACKAGE_BASE_VERSION)
 ARCHS = armv7 arm64
+TARGET = iphone:clang:latest:6.0
 
-include theos/makefiles/common.mk
+THEOS_DEVICE_IP = localhost -p 2222
```

Comment:
Don't add THEOS_DEVICE_IP here. Its very environment dependent, and not really suitable for production code.